neuIPs/neuIP-3.md

```diff
@@ -0,0 +1,53 @@
+---
+eip: 3
+title: Music NFT Release Event
+description: A minimal primitive to emit URI events for social media.
+author: Kevin Neaton (@neatonk), Tim Daubenschütz (@TimDaub)
+discussions-to: https://github.com/neume-network/neuIPs/issues/4
+status: Draft
+created: 2022-09-16
+---
+
+## Abstract
+
+We introduce a minimal on-chain primitive to emit pointers to music NFTs using Ethereum's event log infrastructure.
+
+## Motivation
+
+The modern social network's base layer is distributing content behind Universal Resource Identifiers as defined by RFC 3986. Any web resource can be identified by an URI or a URL and so upgrading Ethereum with a standard way of distributing content is useful.
+
+## Specification
+
+```solidity
+contract Feed {
+  event NewTrack(address indexed curator, string indexed uri);
+  function release(string calldata uri) external {
+    emit NewTrack(msg.sender, uri);
```

Comment:
Maybe we could have a chance here for a user to emit a JSON schema standard for music NFTs.

E.g. we start identifying different semantically versioned schemas as e.g.:

- catalog schema v0.0.1
- neume-network-schema 0.0.2
- etc.

and then whoever is emitting can specific the schema too. Albeit, if the schema is then later changed because the tokenURI is mutable, then this may be a bit ephemeral.

Reply:
After thinking some more about this, we should ideally just ask the event emitter to format their music NFT tokenURI metadata such that we have to do as little transforming as possible